assets/js/src/form_editor/store/actions.jsx

```diff
@@ -45,3 +36,13 @@ export function removeNotice(id) {
     id,
   };
 }
+
+export function* saveForm() {
+  yield {
+    type: 'SAVE_FORM_STARTED',
+  };
+  yield removeNotice('save-form');
```

Comment:
I am not sure if it is a good idea to have this here. Why does `saveForm` need to know anything about notices and if they should or should not be removed? Should something else react on `SAVE_FORM_STARTED` action and hide notice if needed?

Reply:
Good point. It would be probably better to move this to a reducer.